source/best-practices/backup.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+#
+# 使用 rsync 命令备份指定目录
+#
+#        作者： 姚家园、田冬冬
+# 最近修改日期： 2021年2月2日
+#
+
+source="${HOME}/workspace/source"  # 源目录
+backup="${HOME}/workspace/backup"  # 备份目录（通常为移动硬盘的挂载目录）
```

Comment:
Use more realistic directories?
```suggestion
source="${HOME}"                   # 源目录
backup="/mnt/seismo-learn/backup"  # 备份目录（通常为移动硬盘的挂载目录）
```

Reply:
Will revise it after we finish this script.